src/modelfree/multi/score.py

```diff
@@ -130,7 +134,11 @@ def extract_data(path_generator, out_dir, experiment_dirs, ray_upload_dir):
         experiment_root = osp.join(ray_upload_dir, experiment_dir)
         # video_root contains one directory for each score_agent trial.
         # These directories have names of form score-<hash>_<id_num>_<k=v>...
-        for trial_name in os.listdir(experiment_root):
+        for dir_entry in os.scandir(experiment_root):
+            if not dir_entry.is_dir():
```

Comment:
Just out of curiosity, was this change occasioned by the output structure changing as a result of the upgrade, or did you just think this was better overall independent of upgrade?

Reply:
There's now a JSON file in this directory, so this check is needed whereas before it wasn't. (But yes, I should have written it this way in the first place to be defensive.)